examples/hosts/iframe-host/src/inner.ts

```diff
@@ -3,8 +3,12 @@
  * Licensed under the MIT License.
  */
 import { fluidExport as TodoContainer } from "@fluid-example/todo";
-import { Container, Loader } from "@fluidframework/container-loader";
+import { Container } from "@fluidframework/container-loader";
 import { IFluidObject } from "@fluidframework/core-interfaces";
+import {
+    getContainer,
+    InsecureTinyliciousUrlResolver,
+ } from "@fluidframework/get-tinylicious-container";
```

Comment:
the inner shouldn't have any real drivers, as we want to keep the iframe code isolated, and not have to give it tokens or anything